cmd/kindnetd/util.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"net"
+	"strings"
+)
+
+// IsIPv6 returns true if the address is ipv6
+func IsIPv6(address string) bool {
```

Comment:
just found that the net library already has this function, we can avoid this file https://github.com/kubernetes/utils/blob/c55fbcfc754a5b2ec2fbae8fb9dcac36bdba6a12/net/net.go#L118

Reply:
yeah, this is also nice because it works for CIDRs or URLs or addresses afaik.

Reply:
it does not work for CIDRS :man_facepalming:

Reply:
https://github.com/kubernetes-sigs/kind/pull/650